app/controllers/sign_out_controller.rb

```diff
@@ -3,6 +3,11 @@ class SignOutController < ApplicationController
 
   def destroy
     analytics.logout_initiated(method: 'cancel link')
+    irs_attempts_api_tracker.logout_initiated(
+        user_uuid: current_user.present? ? current_user.uuid : "",
```

Comment:
probably preferable to do:
```suggestion
        user_uuid: current_user&.uuid,
```

Reply:
Thank you for this!  It ended up being moot, because we were deleting the property.